packages/keyring-controller/src/KeyringController.ts

```diff
@@ -2358,7 +2466,9 @@ export class KeyringController extends BaseController<
       const serializedKeyrings = await this.#getSerializedKeyrings();
 
       if (
-        !serializedKeyrings.some((keyring) => keyring.type === KeyringTypes.hd)
+        !serializedKeyrings.some(
+          (keyring) => keyring.type === (KeyringTypes.hd as string),
```

Comment:
Similar to other comments, can we remove this cast here?

Reply:
yes, but then the linter is complaining (with a warning).